config/config_mgmt.py

```diff
@@ -153,6 +153,24 @@ def sysLog(self, logLevel=syslog.LOG_INFO, msg=None, doPrint=False):
 
         return
 
+    def _filterTablesInConfig(self, config):
```

Comment:
PEP8 style guidelines state that function names should be `lowercase_with_underscores()`, but since this was missed in the original commit of this file, we will keep this as-is as it matches the rest of the file. However, a refactoring pass should be made to align this file's style with PEP8  and thus the rest of the SONiC code base.

Reply:
Sure, Thx for info, I will try to fix 1084 whenever possible for sure.